src/tools/rustc-workspace-hack/Cargo.toml

```diff
@@ -71,16 +71,20 @@ features = [
 ]
 
 [dependencies]
-bstr = { version = "0.2.13", features = ["default"] }
+bstr = { version = "0.2.17", features = ["default"] }
 byteorder = { version = "1", features = ['default', 'std'] }
 clap = { version = "3.1.1", features = ["derive", "clap_derive"]}
 curl-sys = { version = "0.4.13", features = ["http2", "libnghttp2-sys"], optional = true }
 crossbeam-utils = { version = "0.8.0", features = ["nightly"] }
 libc = { version = "0.2.79", features = ["align"] }
 # Ensure default features of libz-sys, which are disabled in some scenarios.
 libz-sys = { version = "1.1.2" }
+
+# looks like the only user of deprecated `use_std` feature is `combine`, so this
+# can be removed if/when https://github.com/Marwes/combine/pull/348 be merged and released.
+memchr = { version = "2.5", features = ["std", "use_std"] }
 # same for regex
```

Comment:
Nitpick: This comment is now left with an invalid reference! It previously referred to "the previous comment" which spoke about the default features of libz-sys, but you added a comment in between the two and now it looks like "same for regex" means "use_std is deprecated ..."

Reply:
Looks like it.

Reply:
So, could someone please file a follow-up PR to fix that comment?

Reply:
Sure. I assigned #100070 to you.